packages/process/_process.pony

```diff
@@ -347,6 +347,7 @@ primitive _WaitPidStatus
 class _ProcessWindows is _Process
   let hProcess: USize
   let processError: (ProcessError | None)
+  var finalWaitResult: (_WaitResult | None) = None
```

Comment:
per the style guide, we use snake case for fields.

I'll open an issue for that.

https://github.com/ponylang/ponyc/blob/main/STYLE_GUIDE.md#naming

Reply:
opened: https://github.com/ponylang/ponyc/issues/3818